source/configuration_manual/namespace/index.rst

```diff
@@ -32,6 +32,182 @@ you to update an existing namespace (like how ``15-mailboxes.conf`` does) or
 have userdb override namespace settings for specific users
 ``namespace/sectionname/prefix=foo/``.
 
+All namespace settings with defaults::
+
+  namespace inbox {
+    disabled = no
+    hidden = no
+    ignore_on_failure = no
+    inbox = no
+    list = yes
+    location =
+    order = 0
+    prefix =
+    separator =
+    subscriptions = yes
+    type = private
+  }
+
+
+.. _setting-namespace_name:
+
+``namespace name``
+------------------
+
+- Default: <empty>
+- Values: :ref:`string`
+
+Name of the namespace.
+Giving name is optional, but doing so enables referencing the configuration later on.
+
+
+.. _setting-namespace_disabled:
+
+``namespace/disabled``
+----------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace is disabled, and cannot be accessed by user in any way.
+
+
+.. _setting-namespace_hidden:
+
+``namespace/hidden``
+--------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` namespace will be hidden from user.
+It is still accessible if folders are directly opened, or referenced by other components.
+Usually used for lazy expunge or virtual folders.
+
+
+.. _setting-namespace_ignore_on_failure:
+
+``namespace/ignore_on_failure``
+-------------------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+If namespace :ref:`setting-namespace_location` fails to load, this namespace will not be included.
+
+
+.. _setting-namespace_inbox:
+
+``namespace/inbox``
+-------------------
+
+ - Default: `no`
+ - Values: ref:`boolean`
+
+When ``yes`` this namespace will be considered the one holding ``INBOX`` folder.
+There can be only one namespace like this.
+
+
+.. _setting-namespace_list:
+
+``namespace/list``
+------------------
+
+ - Default: `yes`
+ - Values: ref:`boolean`
+
+Whether to include this namespace in LIST output.
```

Comment:
Whether to include this namespace in LIST output when listing its parent folders. It's still possible to LIST the namespace's folders by explicitly asking for them. For example with namespace prefix "lazy-expunge/" using `LIST "" *` won't list it, but using `LIST "" lazy-expunge/*` lists all folders under it.